pkg/storage/replica.go

```diff
@@ -1877,7 +1878,8 @@ func (r *Replica) tryAddWriteCmd(
 			// AmbiguousResultError indicates to caller that the command may
 			// have executed.
 			if tryAbandon() {
-				log.Warningf(ctx, "context cancellation of command %s", ba)
+				log.Warningf(ctx, "context cancellation after %s of attempting command %s",
+					time.Since(startTime), ba)
```

Comment:
I often print durations using `%0.1fs` and `time.Since(startTime).Seconds()`. This keeps a standard duration which makes grepping and sorting logs easier.